src/UWP/ArcGISRuntime.UWP.Viewer/ArcGISRuntime.UWP.Viewer.csproj

```diff
@@ -1250,14 +1268,16 @@
       <Version>100.3.0</Version>
     </PackageReference>
     <PackageReference Include="Esri.ArcGISRuntime.UWP">
-      <Version>100.3.0</Version>
+      <Version>100.3.0-daily2102</Version>
```

Comment:
We need to remove references to the daily builds in both this csproj and the WPF version.